packages/annotorious-core/src/model/Annotation.ts

```diff
@@ -12,16 +12,28 @@ export interface Annotation {
 
     [key: string]: any;
 
-  }
+  };
 
 }
 
-export interface AnnotationTarget {
+export interface AnnotationTarget extends AnnotationLifecycleInfo {
```

Comment:
Is it really necessary to introduce this as another concept? TBH looks a bit like overkill to me.

Reply:
It was convenient to extract the properties of the lifecycle into another type for grouping and use it as the return value for the parse/serialize functions. Additionally, the types of the `AnnotationLifecycleInfo` & `W3CAnnotationLifecycleInfo` will be different, because the first one uses `Date` for the timestamps, and the second one uses `string`

Reply:
Another way to group the lifecycle properties could be adding a type with a `Pick`. So it could look like this:
```ts
export interface AnnotationTarget {...}

type AnnotationLifecycleInfo = Pick<AnnotationTarget, 'creator' | 'created'>;
```
But it seems odd and inversed to me 🤔